apps/session-manager/package.json

```diff
@@ -0,0 +1,44 @@
+{
+    "name": "@pagopa/io-session-manager",
+    "version": "0.0.1",
+    "description": "API for IO session",
+    "main": "./dist/main.js",
+    "license": "MIT",
+    "scripts": {
+        "clean": "shx rm -rf dist",
+        "clean:generated": "shx rm -rf ./src/generated",
+        "build": "tsc",
+        "build:watch": "yarn build -w",
+        "typecheck": "yarn build --noemit",
+        "start": "node dist/server.js",
+        "start:watch": "yarn start watch",
+        "lint": "yarn lint:check --fix",
+        "lint:check": "eslint . -c .eslintrc.js --ext .ts,.tsx --cache",
+        "test": "vitest run",
+        "test:watch": "vitest"
+    },
+    "devDependencies": {
+        "@pagopa/typescript-config-node": "*",
+        "@types/express": "^4.17.21",
+        "@types/node": "~20.10.0",
+        "dependency-check": "^4.1.0",
```

Comment:
at the moment this is not used. should we add a `postbuild` script as follows? `"postbuild": "dependency-check package.json --no-dev --missing ./dist/**/*.js",`

Reply:
Removed for now in [[#IOPID-1763] add eslint + fix based on code review](https://github.com/pagopa/io-auth-n-identity-domain/pull/1/commits/95787ff45a3ddbfeccbb0f616623810c957e883f)